druid-shell/src/platform/gtk/window.rs

```diff
@@ -47,6 +47,9 @@ use super::dialog;
 use super::menu::Menu;
 use super::util;
 
+/// When the GTK reported DPI differs from this we will scale coordinates to achieve it.
+const SCALE_TARGET_DPI: f64 = 96.0;
```

Comment:
I think it would be good to add a not that this is the value reported by GTK (or Windows below) when the scale is 1x in both directions. At least that's how I interpret this constant.